internal/mode/static/telemetry/collector.go

```diff
@@ -187,6 +187,15 @@ func collectGraphResourceCount(
 	return ngfResourceCounts, nil
 }
 
+func computeRouteCount(routes map[graph.RouteKey]*graph.L7Route) (httpRouteCount int64) {
```

Comment:
```suggestion
func computeHTTPRouteCount(routes map[graph.RouteKey]*graph.L7Route) (httpRouteCount int64) {
```

Reply:
Or this function eventually returns a struct with both route counts as separate vars.

Reply:
> Or this function eventually returns a struct with both route counts as separate vars.

@sjberman That's what I presume we'll do, which is why I kept the name generic